packages/common/src/utils/TypeUtil.ts

```diff
@@ -111,7 +111,14 @@ type BalanceQuantity = {
   numeric: string
 }
 
-export type SIWEStatus = 'uninitialized' | 'ready' | 'loading' | 'success' | 'rejected' | 'error'
+export type SIWEStatus =
+  | 'uninitialized'
+  | 'ready'
+  | 'loading'
+  | 'success'
+  | 'rejected'
+  | 'error'
+  | 'authenticating'
```

Comment:
how is it different than loading?

Reply:
authenticating is specific to 1ClickAuth